api/src/main/java/edu/utexas/tacc/tapis/files/api/binders/ServiceJWTCacheFactory.java

```diff
@@ -0,0 +1,40 @@
+package edu.utexas.tacc.tapis.files.api.binders;
+
+import edu.utexas.tacc.tapis.files.api.providers.FileOpsAuthzSystemPath;
+import edu.utexas.tacc.tapis.files.lib.config.IRuntimeConfig;
+import edu.utexas.tacc.tapis.files.lib.config.RuntimeSettings;
+import edu.utexas.tacc.tapis.shared.exceptions.TapisException;
+import edu.utexas.tacc.tapis.sharedapi.security.ServiceJWT;
+import edu.utexas.tacc.tapis.sharedapi.security.ServiceJWTParms;
+import edu.utexas.tacc.tapis.tokens.client.TokensClient;
+import org.glassfish.hk2.api.Factory;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import javax.inject.Inject;
+
+public class ServiceJWTCacheFactory implements Factory<ServiceJWT> {
+    private Logger log = LoggerFactory.getLogger(ServiceJWTCacheFactory.class);
+
+    private IRuntimeConfig runtimeConfig = RuntimeSettings.get();
+
+    @Override
+    public ServiceJWT provide() {
+        try {
+            ServiceJWTParms params = new ServiceJWTParms();
+            params.setServiceName("files");
+            params.setTenant("master");
+            params.setTokensBaseUrl("https://master.develop.tapis.io");
```

Comment:
Can we have this baseURL somewhere in a config/properties file?  Later when we try to change it we have to change it in the code then.